const.go

```diff
@@ -117,6 +152,7 @@ const (
 	// It's not an implementation choice, the value defined in the specification.
 	initialStreamWindow = 256 * 1024
 	maxStreamWindow     = 16 * 1024 * 1024
+	goAwayWaitTime      = 5 * time.Second
```

Comment:
I'd make this_much shorter. E.g., 100ms. We shouldn't need to wait on a round-trip here.